go.spec

```
@@ -54,18 +54,47 @@ any older distro-provided versions.
 %build
 
 %install
+%if (0%{?suse_version} > 0)
+%{__mkdir_p} %{buildroot}/%{_exec_prefix}/lib64/go/%{_go_rel}
+cp -a bin %{buildroot}/%{_exec_prefix}/lib64/go/%{_go_rel}/
+cp -a pkg %{buildroot}/%{_exec_prefix}/lib64/go/%{_go_rel}/
+cp -a src %{buildroot}/%{_exec_prefix}/lib64/go/%{_go_rel}/
+%else
 %{__mkdir_p} %{buildroot}/%{_exec_prefix}
 cp -a bin %{buildroot}/%{_exec_prefix}
 cp -a pkg %{buildroot}/%{_exec_prefix}
 cp -a src %{buildroot}/%{_exec_prefix}
+%endif
 
 %files
+%if (0%{?suse_version} > 0)
+%{_exec_prefix}/lib64/go/%{_go_rel}/bin/*
+%{_exec_prefix}/lib64/go/%{_go_rel}/pkg/*
+%{_exec_prefix}/lib64/go/%{_go_rel}/src/*
+%else
 %{_bindir}/*
 %{_exec_prefix}/pkg/*
 %{_exec_prefix}/src/*
+%endif
+
 %doc
 
+%if (0%{?suse_version} > 0)
+%post
+update-alternatives \
+  --install /usr/bin/go go /usr/lib64/go/%{version}/bin/go $((20+$(echo %{version} | cut -d. -f2))) \
+  --slave /usr/bin/gofmt gofmt /usr/lib64/go/%{version}/bin/gofmt
```

You should use the macros:

```suggestion
  --install %{_bindir}/go go %{_libdir}/go/%{version}/bin/go $((20+$(echo %{version} | cut -d. -f2))) \
  --slave %{_bindir}/gofmt gofmt %{_libdir}/go/%{go_label}/bin/gofmt
```

I'm surprised `rpmlint` is not complaining about that.

---

fixed as you suggested. Thank you!